mule-apikit-module/src/test/resources/artifacts/resource-classloading/src/main/resources/catch-phrase.txt

```diff
@@ -0,0 +1 @@
+You know nothing, Jon Snow.
```

Comment:
?????

Reply:
Copied from the mule runtime repository, do you want me to change it ?

Reply:
If we need this txt file keep it